docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-windows-defender-exclusions-added-via-powershell.asciidoc

```diff
@@ -0,0 +1,145 @@
+[[prebuilt-rule-0-16-2-windows-defender-exclusions-added-via-powershell]]
+=== Windows Defender Exclusions Added via PowerShell
```

Comment:
```suggestion
=== Microsoft Defender Exclusions Added via PowerShell
```

Reply:
Adding this in case it needs to be reflect the changes @joepeeples suggested here:
https://github.com/elastic/security-docs/pull/1922#discussion_r862011985